src/sonic-config-engine/tests/common_utils.py

```diff
@@ -62,22 +61,29 @@ def validate(self, argument):
             parser.add_argument("-p", "--port-config", help="port config file, used with -m or -k", nargs='?', const=None)
             parser.add_argument("-S", "--hwsku-config", help="hwsku config file, used with -p and -m or -k", nargs='?', const=None)
             parser.add_argument("-j", "--json", help="additional json file input, used with -p, -S and -m or -k", nargs='?', const=None)
-            args, unknown = parser.parse_known_args(shlex.split(argument))
+            parser.add_argument("-o", "--output-file", help="Output file", nargs='?', const=None)
```

Comment:
>add_argument [](http://example.com/codeflow?start=19&length=12)

Why change behavior?

Reply:
Because there're some commands that have output redirection.
Original command: https://github.com/sonic-net/sonic-buildimage/blob/f34ca2b6a60f753e1983b552250547866fa10ed6/src/sonic-config-engine/tests/test_j2files.py#L98
Changed command https://github.com/sonic-net/sonic-buildimage/blob/ce3e0759543cd00a5a515b31ffb3e87de48c2e76/src/sonic-config-engine/tests/test_j2files.py#L109
Implementation of new behavior:
https://github.com/sonic-net/sonic-buildimage/blob/ce3e0759543cd00a5a515b31ffb3e87de48c2e76/src/sonic-config-engine/tests/test_j2files.py#L46-#L59

Reply:
I prefer following subprocess functions, adding `stdout=None, stderr=None` to this function parameter list.